flink/json-jobmanager.yml

```diff
@@ -0,0 +1,224 @@
+    path: $.[?(@.id=Status.JVM.Memory.Heap.Committed)]
+    metric:
+      - name: jobmanager.heap.committed
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: committed heap
+        description: The amount of heap memory guaranteed to be available to the JVM
+
+  - name: jobmanager_nonheap_max
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.NonHeap.Max)]
+    metric:
+      - name: jobmanager.nonheap.max
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: used heap
+        description: The maximum amount of non-heap memory that can be used for memory management
+
+  - name: jobmanager_nonheap_used
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.NonHeap.Used)]
+    metric:
+      - name: jobmanager.nonheap.used
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: used non-heap
+        description: The amount of non-heap memory currently used
+
+  - name: jobmanager_nonheap_committed
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.NonHeap.Committed)]
+    metric:
+      - name: jobmanager.nonheap.committed
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: committed non-heap
+        description: The amount of non-heap memory guaranteed to be available to the JVM
+
+  - name: jobmanager_direct_max
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Direct.TotalCapacity)]
+    metric:
+      - name: jobmanager.direct.max
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: direct used
+        description: The total capacity of all buffers in the direct buffer pool
+
+  - name: jobmanager_direct_used
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Direct.MemoryUsed)]
+    metric:
+      - name: jobmanager.direct.used
+        source: $.[?(@.id=Status.JVM.Memory.Direct.MemoryUsed)].value
+        type: long_gauge
+        unit: bytes
+        label: direct used
+        description: The amount of memory used by the JVM for the direct buffer pool
+
+  - name: jobmanager_direct_count
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Direct.Count)]
+    metric:
+      - name: jobmanager.direct.count
+        source: value
+        type: long_gauge
+        label: direct count
+        description: The number of buffers in the direct buffer pool
+
+  - name: jobmanager_mapped_max
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Mapped.TotalCapacity)]
+    metric:
+      - name: jobmanager.mapped.max
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: mapped count
+        description: The number of buffers in the mapped buffer pool
+
+  - name: jobmanager_mapped_used
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Mapped.MemoryUsed)]
+    metric:
+      - name: jobmanager.mapped.used
```

Comment:
Rename, and its label. Same for other mapped below.

Reply:
rename to jobmanager.memory.mapped.buffers.used